A blocking bug is found for channel:  
	There are 1 goroutines blocked at this channel
goroutine 18 [running]:
runtime/debug.Stack(0x30, 0x0, 0x0)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x9d
runtime/debug.PrintStack()
	/usr/local/go/src/runtime/debug/stack.go:16 +0x22
gotest/gooracle.ReportBug(0xc0000c6240)
	/data/ziheng/shared/gotest/gotest/src/gotest/gooracle/gooracle.go:219 +0x1a4
gotest/gooracle.(*ChanInfo).CheckBlockBug(0xc0000c6240)
	/data/ziheng/shared/gotest/gotest/src/gotest/gooracle/gooracle.go:145 +0x426
gotest/gooracle.TestGLDetect(0xc0001f0120)
	/data/ziheng/shared/gotest/gotest/src/gotest/gooracle/gooracle_test.go:37 +0x1c1
testing.tRunner(0xc0001f0120, 0x55a098)
	/usr/local/go/src/testing/testing.go:991 +0xdc
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1042 +0x357
--- FAIL: TestGLDetect (0.00s)
panic: 1 [recovered]
	panic: 1

goroutine 18 [running]:
testing.tRunner.func1.1(0x51c4e0, 0x578618)
	/usr/local/go/src/testing/testing.go:940 +0x2f5
testing.tRunner.func1(0xc0001f0120)
	/usr/local/go/src/testing/testing.go:943 +0x3f9
panic(0x51c4e0, 0x578618)
	/usr/local/go/src/runtime/panic.go:969 +0x166
gotest/gooracle.ReportBug(0xc0000c6240)
	/data/ziheng/shared/gotest/gotest/src/gotest/gooracle/gooracle.go:220 +0x1c0
gotest/gooracle.(*ChanInfo).CheckBlockBug(0xc0000c6240)
	/data/ziheng/shared/gotest/gotest/src/gotest/gooracle/gooracle.go:145 +0x426
gotest/gooracle.TestGLDetect(0xc0001f0120)
	/data/ziheng/shared/gotest/gotest/src/gotest/gooracle/gooracle_test.go:37 +0x1c1
testing.tRunner(0xc0001f0120, 0x55a098)
	/usr/local/go/src/testing/testing.go:991 +0xdc
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1042 +0x357
exit status 2
FAIL	gotest/gooracle	0.012s
